### PR TITLE
Fixed error - int() on None type

### PR DESCRIPTION
**Pull request by mikeclark**
**Description of the change**:
Checking for None first before running int() on download counts.  Wanted to be able to record null, 0, and >0 counts in the db.

**Reason for the change**:
Came across int() must be string or int not 'NoneType' error(s) when running with only one package_manager_urls in config.yml 

### Checklist

Make sure all of these items are complete, or else the PR will be ineligible for a code review.

- [x] Code passes all existing [tests](https://github.com/sendgrid/open-source-library-data-collector/tree/master/test)
- [x] Any new functionality added includes new unit tests in [`tests/test.py`](https://github.com/sendgrid/open-source-library-data-collector/blob/master/test/test.py)
- [x] Create or update example code to show the new functionality in action.
- [x] All code, branch, and git naming and style conventions are followed (see [`CONTRIBUTING.md`](https://github.com/sendgrid/open-source-library-data-collector/blob/master/CONTRIBUTING.md#style-guidelines--naming-conventions))
- [x] Feature branch has been rebased off of the latest `master` branch. ( see [`CONTRIBUTING.md`](https://github.com/sendgrid/open-source-library-data-collector/blob/master/CONTRIBUTING.md#creating-a-pull-request) ).

If you have questions, please send an email [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/open-source-library-data-collector?pullRequest=76) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/open-source-library-data-collector?pullRequest=76) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/open-source-library-data-collector?pullRequest=76) it.</sub>

File: package_managers.py [hunk 1]
```diff
@@ -189,6 +189,21 @@ def update_db(
...
+            num_ruby_downloads,
+            num_python_http_client_downloads,
+            num_python_open_source_library_data_collector_downloads,
+            num_ruby_http_client_downloads,
+            num_csharp_http_client_downloads,
+            num_php_http_client_downloads,
+            num_node_http_client_downloads]
+
+        for parm in parms:
+            parm = None if parm is None else int(parm)
```
Comment (mbernier) [inline package_managers.py]: Is python passing by reference here or is this creating a new parameter called `parm` ?

**Comment by mikeclark** [comment]
Passing by reference.  Wasn't sure if that was the best way to do it (I just didn't want to wrap each line in the same if statement) or whether I should be looking through just the passed in args using something like locals

**Comment by mbernier** [comment]
Thanks!

**Comment by thinkingserious** [comment]
Hello @mikeclark,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**